docs/content/dagster-cloud/account/managing-user-agent-tokens.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Managing user and agent tokens in Dagster Cloud | Dagster Docs
+
+noindex: True
+---
+
+# Managing user and agent tokens in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+In this guide, we'll walk you through creating and revoking user and agent tokens in Dagster Cloud.
+
+---
+
+## Managing agent tokens
+
+<Note>
+  <strong>Editor</strong>, <strong>Admin</strong>, or{" "}
+  <strong>Organization Admin</strong> permissions are required to manage agent
+  tokens. This includes viewing existing tokens.
```

Comment:
Could you clarify? This callout essentially says "You need to be one of these user roles to manage tokens," not that you have to have all three.

These not being deployment-scoped could be clarified here, but I'm otherwise not sure what's incorrect about that statement.